lib/srv/db/cloud/watchers/watcher.go

```diff
@@ -167,7 +167,7 @@ func makeFetchers(ctx context.Context, config *WatcherConfig) (result []Fetcher,
 func makeAWSFetchers(clients cloud.Clients, matchers []services.AWSMatcher) (result []Fetcher, err error) {
 	type makeFetcherFunc func(cloud.Clients, string, types.Labels) (Fetcher, error)
 	makeFetcherFuncs := map[string][]makeFetcherFunc{
-		services.AWSMatcherRDS:         {makeRDSInstanceFetcher, makeRDSAuroraFetcher},
+		services.AWSMatcherRDS:         {makeRDSInstanceFetcher, makeRDSAuroraFetcher, makeRDSProxyFetcher},
```

Comment:
Wonder if we should consider making the makeRDSProxyFetcher as a separate matcher like `rdsproxy `  and make the discovery flow independed from current `rds` auto discovery flow .

I can imagine a scenario where a customer uses RDS Proxy access with some default tags like '*' 
without enabling IMA auth on RDS instance.  By default teleport agent will show RDS Proxy with RDS instances  where access though RDS instances will be never used.

Reply:
Yes. Totally agreed. There is no point to have both RDS and RDS Proxy for the same database. I actually started with a separate matcher but removed it at some point. I will add it back.